contracts/src/v0.8/keystone/KeystoneForwarder.sol

```diff
@@ -57,9 +58,9 @@ contract KeystoneForwarder is IForwarder, ConfirmedOwner, TypeAndVersionInterfac
   // reportId = keccak256(bytes20(receiver) | workflowOwner | workflowExecutionId)
   mapping(bytes32 reportId => DeliveryStatus status) internal s_reports;
 
-  event ReportDelivered(
+  event ReportProcessed(
     address indexed receiver,
-    bytes32 indexed workflowOwner,
+    address indexed workflowOwner,
```

Comment:
Is the owner actually an ethereum address? Or is it just some private key? I didn't want to make assumptions here since this also won't be the same `address` type across all chains

Reply:
I assume 20 bytes gives us enough space to support the "owner" concept more generally. Nothing prevents us from having a different type for other chains as long as it can fit into 20 bytes. We can also pad 🤷‍♂️

I didn't want to add gas costs unnecessarily for the EVM chains (happy path).

Reply:
I'd type it as byte20 then to not imply an actual ethereum address -- for me that usually means a wallet or an on chain contact